docs/spec/spec.md

```diff
@@ -249,15 +249,15 @@ public function main() returns error? {
 }
 ```
 
-## 4.3. Client Streaming RPC
+## 4.3. Client streaming RPC
```

Comment:
```suggestion
## 4.3. Client streaming RPC
```
Should this be `RPC client streaming`?

Reply:
This is the correct term https://grpc.io/docs/what-is-grpc/core-concepts/#rpc-life-cycle